### ISSUE TITLE
Detector Tubes will stick in the "on" position when the mapblocks containing them fall out of memory

### ISSUE THREAD

**Issue by dontfwiththestream** [minetest-mods/mesecons#654]
Minetest server version 5.7.0

Expected behavior:  in the absence of an activator mesecon activator,  mesecons are inactive 

Actual behavior: 
![image](https://github.com/minetest-mods/mesecons/assets/35442128/8013f309-d00e-450a-b928-34dd45add0b1)

inside the small pipeworks loop is a node, its just not visible inside the tube because its grating.  This simple machine should turn on the lights in sequence, then turn them off. The state it is in shouldn't be "possible"

**Comment by dontfwiththestream** [comment]
I believe the behavior will only occur when the area is not loaded into memory. In the initial replication of the bug above, i neglected to mention that the mesecons only entered the invalid state as shown after I had disconnected my client from the server, and later rejoined. I replicated the machine with an admin world anchor from techic to forceload the area and tried to replicate the bug by disconnecting from the server again, and the duplicate machine behaved as intended. 

Although technic world anchors appear to be a workaround for this issue, it is a poor solution because thats a rather large dependency to make mesecons function correctly. 

I will continue experimenting and see if I can nail down anything more specific about this bug
